paymentsheet/src/main/java/com/stripe/android/paymentsheet/verticalmode/PaymentMethodVerticalLayoutUI.kt

```diff
@@ -172,7 +172,7 @@ private fun EditButton(onClick: () -> Unit) {
     Text(
         stringResource(id = com.stripe.android.R.string.stripe_edit),
         color = MaterialTheme.colors.primary,
-        style = MaterialTheme.typography.button,
+        style = MaterialTheme.typography.subtitle1,
```

Comment:
It feels like a bug that this fixes the issue. It seems like `button` doesn't respect the theme properly. Updating compose doesn't fix it though, so if it is a bug, it seems like an unfixed one

Reply:
Should we check with design to ensure they're ok with the way this looks?

Reply:
checking now! Will re-request review if I make any updates

Reply:
Didn't get any feedback, but updated the font weight a bit so that the visual diff here is smaller!